engine/src/main/java/com/datatorrent/stram/plan/logical/DAGSetupPluginManager.java

```diff
@@ -145,8 +150,7 @@ public void dispatch(DAGSetupEvent event)
 
   public static synchronized DAGSetupPluginManager getInstance(Configuration conf)
```

Comment:
@tushargosavi @chinmaykolhatkar @PramodSSImmaneni Does the method need to be synchronized?